src/app/functions/server/changeSubscription.ts

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Subscriber } from "../../(nextjs_migration)/(authenticated)/user/breaches/breaches";
+import { updateFxAProfileData } from "../../../db/tables/subscribers";
+
+const MONITOR_PREMIUM_CAPABILITY = "monitor";
+
+export async function changeSubscription(
```

Comment:
@Vinnl I moved this to a server function instead of spreading it around the code, hopefully is easier to read now. 

The reason I was thinking the `Set` conversion was easier to read is that the `subscriptions` key may not exist, and we need to create it if it does not.